src/config/config.go

```diff
@@ -276,3 +320,21 @@ func isCompatibleComponent(component types.ZarfComponent, filterByOS bool) bool
 
 	return validArch && validOS
 }
+
+func promptVariable(variable types.ZarfPackageVariable) string {
+	var value string
+
+	pterm.Println()
+
+	prompt := &survey.Input{
+		Message: "Please provide a value for '" + variable.Name + "'",
+	}
+
+	if variable.Default != nil {
+		prompt.Default = *variable.Default
+	}
+
+	_ = survey.AskOne(prompt, &value)
```

Comment:
We probably shouldn't ignore the error here. A `ctrl+c` would return some form of an `&errors.errorString{s:"interrupt"}` error. Since the err is getting ignored this leads to the deployment continuing with the prompted value just being left as an empty string.

Reply:
Fixed